apps/remix-ide/src/app/tabs/testTab/testTab.js

```diff
@@ -4,6 +4,7 @@ const remixPath = require('path')
 
 class TestTabLogic {
```

Comment:
this file can be removed right?

Reply:
yes, final code cleaning is yet to be done